src/local-history/local-history-manager.ts

```diff
@@ -208,24 +208,34 @@ export class LocalHistoryManager {
     /**
      * Revert the current active editor to one of its previous revision.
      */
-    public async revertToPrevRevision(previous: vscode.TextEditor, current: vscode.TextEditor): Promise<void> {
-        if (current && previous) {
-            const revisionContent: string | undefined = await this.readFile(previous.document.fileName);
-            const activeDocumentContent: string = await this.readFile(current.document.fileName);
-            if (revisionContent && (activeDocumentContent === revisionContent)) {
-                // Prevent user from reverting to the same revision more than once.
-                // Prevent duplicate 'revert change' revision.
-                return;
-            }
+    public async revertToPrevRevision(previous: vscode.Uri): Promise<void> {
+        if (previous) {
+            const current = vscode.window.activeTextEditor!.document.uri;
```

Comment:
It might not be necessary to use the `activeTextEditor`, we can defer the `uri` from `previous` as it is the same (except the `.local-history` part). My worry is if this will somehow not be true at some point.